__tests__/lib/authorize.test.js

```diff
@@ -0,0 +1,20 @@
+const authorize = require('../../__mock__/lib/authorize.mock')
+
+test('test authorize middleware', async () => {
```

Comment:
testes set authorization, tested authorize too. Separately both works fine. What if together they do not work?
 Use same key to set and authorize to make sure together they work as Swiss clock.